src/modules/Lan.rb

```diff
@@ -818,154 +818,76 @@ def UseNetworkManager
 
     def IfcfgsToSkipVirtualizedProposal
       skipped = []
-      Builtins.foreach(LanItems.Items) do |current, _config|
-        ifcfg = Ops.get_string(LanItems.Items, [current, "ifcfg"], "")
-        if NetworkInterfaces.GetType(ifcfg) == "br"
+
+      LanItems.Items.each do |_current, config|
+        ifcfg = config["ifcfg"]
+        ifcfg_type = NetworkInterfaces.GetType(ifcfg)
+
+        case ifcfg_type
+        when "br"
           NetworkInterfaces.Edit(ifcfg)
```

Comment:
Exactly speaking, you need it just because of accessing ```NetworkInterfaces.Current``` a few lines bellow. As this call is incorrect itself (it directly access internal structure of NetworkInterfaces) and use of Edit is misleading here (no editation is done) I propose to use ```LanItems.GetDeviceMap``` instead of ```NetworkInterfaces.Current```. ```NetworkInterfaces.Edit``` can be dropped in such case.